### PR TITLE
Dashboard flex box

### PR DESCRIPTION
**Pull request by acrdlph**
ok, not the cleanest code ever but its working .. will fix linting etc.

### CODE REVIEW

File: src/components/Dashboard/MyToken/MyToken.jsx [hunk 1]
```diff
@@ -119,11 +127,16 @@ class MyTokens extends Component {
...
+    displayName: 'My Token',
+    editingServices: false,
   };
 
 
   async componentDidMount() {
-    const { address, drizzle } = this.props;
+    const { drizzle } = this.props;
+    const address = this.props.match.params.tokenAddress;
+    console.log("biatch")
```
Comment (lsaether) [inline src/components/Dashboard/MyToken/MyToken.jsx]: lol, clean pls

File: src/components/Dashboard/MyToken/MyToken.jsx [hunk 2]
```diff
@@ -375,9 +404,12 @@ class MyTokens extends Component {
 
 
 
-          <Grid item xs={6} sm={6}>
+          <Grid item xs={12} sm={12} md={6}>
```
Comment (lsaether) [inline src/components/Dashboard/MyToken/MyToken.jsx]: the values apply for the specified size and all larger values, so you can shorten this to
`<Grid item xs={12} md={6}>`

since `md` is the breakpoint when you want the grid item to be smaller, `sm` will implicitly have the same value as `xs`

File: src/components/Dashboard/MyToken/MyToken.jsx [hunk 3]
```diff
@@ -375,9 +404,12 @@ class MyTokens extends Component {
...
             <Card className={classes.card}>
               <CardContent>
+              <Typography className={classes.title} color="textSecondary" gutterBottom>
+                      7 day price chart
```
Comment (lsaether) [inline src/components/Dashboard/MyToken/MyToken.jsx]: fix spacing in these two lines

File: src/components/Dashboard/MyToken/MyToken.jsx [hunk 4]
```diff
@@ -403,9 +435,12 @@ class MyTokens extends Component {
               </CardContent>
             </Card>
           </Grid>
-          <Grid item xs={6} sm={6}>
+          <Grid item xs={12} sm={12} md={6}>
```
Comment (lsaether) [inline src/components/Dashboard/MyToken/MyToken.jsx]: same as above

File: src/components/Dashboard/MyToken/MyToken.jsx [hunk 5]
```diff
@@ -403,9 +435,12 @@ class MyTokens extends Component {
...
             <Card className={classes.card}>
               <CardContent>
+              <Typography className={classes.title} color="textSecondary" gutterBottom>
+                      Bonding curve
```
Comment (lsaether) [inline src/components/Dashboard/MyToken/MyToken.jsx]: spacing

File: src/components/Dashboard/MyToken/MyToken.jsx [hunk 6]
```diff
@@ -428,16 +463,16 @@ class MyTokens extends Component {
...
 
             <Card className={classes.card}>
               <CardHeader
 
                 action={
                   <div>
-                    <Button color="secondary" size="sm" onClick={this.makeEditable} style={{ float: 'right' }}>
-                      Edit
+                    <Button color={this.state.editingServices ? 'primary' : 'secondary'} size="sm" onClick={this.toggleServiceEditable} style={{ float: 'right' }}>
+                    {this.state.editingServices ? 'Save' : 'Edit'}
```
Comment (lsaether) [inline src/components/Dashboard/MyToken/MyToken.jsx]: spacing

File: src/components/Dashboard/MyToken/MyToken.jsx [hunk 7]
```diff
@@ -457,8 +492,13 @@ class MyTokens extends Component {
                         id="standard-required"
                         label="req"
                         defaultValue="Service 1"
+                        value={this.state.service1}
+                        onChange={this.handleChange('service1')}
                         className={classes.textField}
                         margin="normal"
+                        InputProps={{
+                      readOnly: !this.state.editingServices,
```
Comment (lsaether) [inline src/components/Dashboard/MyToken/MyToken.jsx]: spacing

File: src/components/Dashboard/MyToken/MyToken.jsx [hunk 8]
```diff
@@ -478,6 +518,9 @@ class MyTokens extends Component {
                         }}
                         helperText="Please select a category"
                         margin="normal"
+                        InputProps={{
+                      readOnly: !this.state.editingServices,
```
Comment (lsaether) [inline src/components/Dashboard/MyToken/MyToken.jsx]: spacing

File: src/components/Dashboard/MyToken/MyToken.jsx [hunk 9]
```diff
@@ -498,6 +541,9 @@ class MyTokens extends Component {
                           shrink: true,
                         }}
                         margin="normal"
+                        InputProps={{
+                      readOnly: !this.state.editingServices,
```
Comment (lsaether) [inline src/components/Dashboard/MyToken/MyToken.jsx]: spacing

File: src/components/Dashboard/MyToken/MyToken.jsx [hunk 10]
```diff
@@ -509,16 +555,15 @@ class MyTokens extends Component {
             </Card>
           </Grid>
 
-          <Grid item xs={8}>
+          <Grid item xs={12} sm={12} md={8}>
```
Comment (lsaether) [inline src/components/Dashboard/MyToken/MyToken.jsx]: clean

File: src/components/Dashboard/MyToken/MyToken.jsx [hunk 11]
```diff
@@ -509,16 +555,15 @@ class MyTokens extends Component {
...
             <Card className={classes.card}>
-              <CardHeader
-                title={"Requests and investments"}
-                subheader={"Honor your token"}
-              />
+              <Typography>
```
Comment (lsaether) [inline src/components/Dashboard/MyToken/MyToken.jsx]: remove

File: src/components/Dashboard/Tab/Tab.jsx [hunk 12]
```diff
@@ -31,14 +32,26 @@ class ScrollableTabsButtonAuto extends React.Component {
...
 
   render() {
     const { classes } = this.props;
     const { value } = this.state;
 
+    const links =
+      this.props.tokens &&
+      this.props.tokens.map(token => {
+        return (
+          // <NavLink to={`/dashboard/#${token.address}`}>{token.name}</NavLink>
```
Comment (lsaether) [inline src/components/Dashboard/Tab/Tab.jsx]: remove dead code

**Comment by lsaether** [comment]
I think running `yarn format` would resolve half of these formatting fixes

File: src/components/Dashboard/MyToken/MyToken.jsx (hunk 1 above)
Comment (acrdlph) [reply src/components/Dashboard/MyToken/MyToken.jsx]: hahahahahaha

**Comment by acrdlph** [comment]
Thanks a lot for review and suggestions!

I fixed these issues and worked more on modularizing the dashboard.

Continuing after lunch with getting the data from IPFS.
